webrtc-stats.html

```diff
@@ -1187,34 +975,23 @@ <h3>
              unsigned long        keyFramesDecoded;
              unsigned long        frameWidth;
              unsigned long        frameHeight;
-             unsigned long        frameBitDepth;
              double               framesPerSecond;
              unsigned long long   qpSum;
              double               totalDecodeTime;
              double               totalInterFrameDelay;
              double               totalSquaredInterFrameDelay;
-             boolean              voiceActivityFlag;
              DOMHighResTimeStamp  lastPacketReceivedTimestamp;
-             double               averageRtcpInterval;
              unsigned long long   headerBytesReceived;
              unsigned long long   fecPacketsReceived;
              unsigned long long   fecPacketsDiscarded;
              unsigned long long   bytesReceived;
-             unsigned long long   packetsFailedDecryption;
```

Comment:
this seems useful for SRTP errors but I am not sure whether the inbound stats are the right place for it

Reply:
Not right place meaning, it's applicable to the transport where it would be all packets not just RTP?

Reply:
the inbound *rtp* stats. One typically wants to drop packets that failed to process without doing further processing such as demuxing / looking up an associated receiver.